reposilite-plugins/webhook-plugin/src/main/kotlin/com/reposilite/plugin/webhook/api/DiscordWebhook.kt

```diff
@@ -0,0 +1,70 @@
+package com.reposilite.plugin.webhook.api
+
+// Core
+import club.minnced.discord.webhook.WebhookClientBuilder;
+// Embed
+import club.minnced.discord.webhook.receive.ReadonlyMessage;
+import club.minnced.discord.webhook.send.WebhookEmbedBuilder
+// Message
+import club.minnced.discord.webhook.send.WebhookMessageBuilder;
+import com.reposilite.maven.Repository
+import com.reposilite.storage.api.Location
+// Config
+
```

Comment:
Avoid comments, especially in such places. It's hard to maintain it, and in the end it does not tell anything useful.